alws/routers/sign_key.py

```diff
@@ -42,7 +42,12 @@ async def create_sign_key(
 ):
     try:
         payload.owner_id = user.id
-        return await sign_key.create_sign_key(db, payload)
+        key = await sign_key.create_sign_key(db, payload)
+        if payload.platform_ids:
+            # reloading the key to load platforms
+            key = await sign_key.get_sign_key(db, payload.keyid)
```

Comment:
You can adjust `db.refresh` at the end of `create_sign_key` func in the following way instead of re-fetching the whole object:
```python
await db.refresh(sign_key, attribute_names=['platforms'])
```
Also, I don't get why we need that here, don't we set `platform_ids` in `create_sign_key` func?

Reply:
Thanks for help, with updated `db.refresh()` it works fine now!

We don't set `platform_ids` in `create_sign_key()` anymore, i removed `platform_ids` field from sqlalchemy model, cause it's no longer neccessary as we have `platforms_sign_keys` table now. As far as i see, the only place where `platform_ids` are used is in payload in response of `sign-keys/new/` endpoint, so i figured that we can just add them directly in the router